bin/yii

```diff
@@ -1,12 +1,18 @@
 #!/usr/bin/env php
 <?php
 
+declare(strict_types=1);
+
+defined('YII_ENV') or define('YII_ENV', 'DEV'); // PROD, DEV
```

Comment:
How would you redefine it?

Reply:
The problem is composer-plug-in adds env variables, or constants to parameters, when variables and constants must be available to the environment.

Reply:
`getenv()`